addons/packages/secretgen-controller/0.8.0/bundle/config/schema.yaml

```diff
@@ -27,3 +27,12 @@ secretgenController:
   namespace: secretgen-controller
   #@schema/desc "Whether to create namespace specified for secretgen-controller"
   createNamespace: true
+  image:
```

Comment:
Please add a comment like `#! Deprecated. Kept for backward compatibility`

Reply:
Fixed